From e530708102631e15f862beaec76288ef68f5dad4 Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Wed, 11 Nov 2020 16:54:18 -0600
Subject: [PATCH 21/22] [webusb] Don't reinitialize a failed connection

This is a hack that should be removed, but it appears that this will
continuously loop over and over again on Linux without udev.
---
 chrome/browser/usb/web_usb_detector.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/usb/web_usb_detector.cc b/chrome/browser/usb/web_usb_detector.cc
index 6a55bca80172f..07b7fd5c81752 100644
--- a/chrome/browser/usb/web_usb_detector.cc
+++ b/chrome/browser/usb/web_usb_detector.cc
@@ -274,7 +274,7 @@ void WebUsbDetector::OnDeviceManagerConnectionError() {
   client_receiver_.reset();
 
   // Try to reconnect the device manager.
-  Initialize();
+  // Initialize();
 }
 
 void WebUsbDetector::SetDeviceManagerForTesting(
-- 
2.26.2

